### PR TITLE
Add zora network

### PR DESCRIPTION
**Pull request by iainnash**

### CODE REVIEW

File: yarn.lock
Comment (wilsoncusack) [inline yarn.lock:1]: we use pnpm :)

**Comment by wilsoncusack** [comment]
thank you sir!

**Comment by kulkarohan** [comment]
updated a few contracts to their proxy addresses, should be good now!

**Comment by wilsoncusack** [comment]
@kulkarohan pls remove yarn.lock :)

**Review by wilsoncusack — APPROVED** [review]

**Pull request merged**